core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -528,13 +529,14 @@ public static boolean isValidTableName(CharSequence tableName, int fsFileNameLim
                 case '\u0006':
                 case '\u0007':
                 case '\u0008':
-                case '\u0009': // Control characters, except \n.
-                case '\u000B': // New line allowed for compatibility, there are tests to make sure it works
+                case '\u0009':
```

Comment:
good question, I meant to remove just the 2nd comment `/ New line allowed for compatibility, there are tests to make sure it works`. Because new line is no longer allowed. The first comment should stay there. Well spotted!